hack/cluster-clean.sh

```diff
@@ -47,29 +43,29 @@ function remove_finalizers() {
         local arr=($p)
         local name="${arr[0]}"
         local ns="${arr[1]}"
-        _kubectl patch vmsnapshots $name -n $ns --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]'
+        patch_remove_finalizers vmsnapshots $name -n $ns
```

Comment:
maybe worth to keep consistency please?
`-n $ns` as the first argument all over
like here
https://github.com/kubevirt/kubevirt/pull/6285/files#diff-918eff3717e4c5a0b1cba2ea435965117616c8ce4cfeb5f8fabb95812a0c480eR36

Reply:
ok, added https://github.com/kubevirt/kubevirt/pull/6285/commits/a1297cb84f68d4366cf151b9eb567d782576b4e8